### PR TITLE
Change styles in TmplParentPointBoxComponent

### PR DESCRIPTION
**Pull request by egor-mir**
PR Checklist

- [x] - Latest `master` branch merged
- [x] - PR title descriptive (can be used in release notes)

## Description
Css changes in TmplParentPointBox Component:
Add scale factor and flex property for display two items per row.
Template [Link](https://plh-teens-app1--pr572-feat-parent-point-bo-l37d6rfp.web.app/template/feature_parent_point_box)
_What this PR does_

## Git Issues

_Closes #_

## Screenshots/Videos
image

_If useful, provide screenshot or capture to highlight main changes_

### CODE REVIEW

**Review by michaelclapham — APPROVED** [review]

File: src/app/shared/components/template/components/points-item/points-item.component.scss [hunk 1]
```diff
@@ -1,10 +1,10 @@
 .wrapper-point {
```
Comment (michaelclapham) [inline src/app/shared/components/template/components/points-item/points-item.component.scss:1]: .wrapper-point {
    --scale-factor--point: 1;
}
@media screen and (max-width: 400px) {
  .wrapper-point {
    --scale-factor--point: 0.227vw; // I think this is the right number 100 vw / 440
  }
}

**Review by michaelclapham — CHANGES_REQUESTED** [review]
Is there a way we could set --scale-factor--point using a CSS media query? It seems like it would be more reliable and efficient that way.

For example
.wrapper-point {
--scale-factor--point: 1;
}
@media screen and (max-width: 400px) {
.wrapper-point {
--scale-factor--point: 0.227vw; // I think this is the right number 100 vw / 440
}
}

Not sure if the media query logic is exactly right, but you get the idea.

File: src/app/shared/components/template/components/points-item/points-item.component.scss (hunk 1 above)
Comment (egor-mir) [reply src/app/shared/components/template/components/points-item/points-item.component.scss:1]: But now it is dynamically and calculates based on template-container width. This is more universal solution from my point of view

Comment (michaelclapham) [reply src/app/shared/components/template/components/points-item/points-item.component.scss:1]: I don't think that's true, but merging anyway

**Review by smborio — APPROVED** [review]
problems sorted, looking good

**Pull request merged**